runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/RemoteBundle.java

```diff
@@ -29,7 +30,7 @@
  * <p>When a RemoteBundle is closed, it will block until bundle processing is finished on remote
```

Comment:
Please add a comment here that RemoteBundles must be closed since this is no longer implied by the `AutoCloseable` type.

Reply:
Not relevant anymore, reverted to AutoCloseable.